### ISSUE TITLE
How you train the models on vox (vox-256.yaml and vox-adv-256.yaml)

### ISSUE THREAD

**Issue by Cold-Winter** [AliaksandrSiarohin/first-order-model#426]
Amazing work! I am trying to reproduce the results of the paper. But i saw two config file for training on voxceleb dataset. I am wondering about the training details since there are not a lot in the paper. Do you first train with vox-256 config (basically only reconstruction loss without gan loss) and then finetune the model with vox-adv-256 config (with gan loss) so that you can generate more realistic images. Please tell me whether my understanding is correct or not.

**Comment by AliaksandrSiarohin** [comment]
Yes this is correct.

**Comment by Cold-Winter** [comment]
Thanks you for the quick reply, I notice that the vox-adv-256.yaml is different from vox-256.yaml. When you do adversarial finetune, do you load the state of optimizer as well? Or just load the pre-trained generator and keypoint detector, and use a random initialized new discriminator (without spectral norm) to train another 150 epoches?

**Comment by AliaksandrSiarohin** [comment]
Both is used. But I double it makes any difference.
